### PR TITLE
Ember 2.13 blog post

### PR DESCRIPTION
**Pull request by locks**
- [Rendered](https://github.com/emberjs/website/blob/e26a2d042c1f731c07b230a6193280cca30d1262/source/blog/2017-04-24-ember-2-13-released.md).
- [Preview](https://deploy-preview-2890--chess-master-boris-50065.netlify.com/blog/2017/04/24/ember-2-13-released.html).

## Ship approvals

- [ ] Ember (@chancancode)
- [x] Data (@bmac)
- [x] CLI (@rwjblue)
- [ ] Website (@locks, @mixonic)

## Checklist

- Ember
  - [ ] Blog
  - [ ] v2.13.0
  - [x] v2.14.0-beta.1
- Ember Data
  - [ ] Blog
  - [ ] v2.13.0
  - [ ] v2.14.0-beta.1
- Ember CLI
  - [ ] Blog
  - [ ] v2.13.0
  - [ ] v2.14.0-beta.1

Team scribes should replace relevant `TBK` with content. See https://github.com/emberjs/website/pull/2824/files for inspiration.

### CODE REVIEW

File: source/blog/2017-04-24-ember-2-13-released.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 2.13
+
+TBK
+
+#### Deprecations in Ember Data 2.13
+
+Ember Data 2.12 contains no new deprecations.
```
Comment (tsing80) [inline source/blog/2017-04-24-ember-2-13-released.md]: 2.12 -> 2.13

File: source/blog/2017-04-29-ember-2-13-released.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+response.
+
+* [401] `DS.UnauthorizedError`
+* [403] `DS.ForbiddenError`
+* [404] `DS.NotFoundError`
+* [409] `DS.ConflictError`
+* [500] `DS.ServerError`
+
+Thanks to [tchak](https://github.com/tchak) and
+[twokul](https://github.com/tchak) for their work on this feature and
```
Comment (tricknotes) [inline source/blog/2017-04-29-ember-2-13-released.md]: Maybe `https://github.com/tchak` -> `https://github.com/twokul` ?

**Comment by rwjblue** [comment]
The main sections of the blog post are finished.  Please review.

File: source/blog/2017-04-29-ember-2-13-released.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 2.13
+
+Ember Data 2.13 represents the work of 20 direct contributors
+and over 120 commits.
+
+The `ds-extended-errors` ([#3586](https://github.com/emberjs/data/pull/3586) [#4287](https://github.com/emberjs/data/pull/4287)) feature has been enabled on the beta branch for Ember Data 2.13.
```
Comment (acorncom) [inline source/blog/2017-04-29-ember-2-13-released.md]: Is this feature enabled only in beta (i.e. 2.14) or is it available now?

File: source/blog/2017-04-29-ember-2-13-released.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+using it due to lack of public API options (though the router service should address these remaining
+cases). Please review the [deprecation guide](https://emberjs.com/deprecations/v2.x/#toc_ember-router-router-renamed-to-ember-router-_routermicrolib)
+for more details.
+
+For more details on the changes in Ember.js 2.13, please review the
+[Ember.js 2.13.0 release page](https://github.com/emberjs/ember.js/releases/tag/v2.13.0).
+
+### Upcoming Changes in Ember.js 2.14
+
+TBK
```
Comment (acorncom) [inline source/blog/2017-04-29-ember-2-13-released.md]: Do we have anything here?

**Pull request merged**